foundation/foundation-mda/src/main/resources/templates/cucumber.test.cdi.context.java.vm

```diff
@@ -26,26 +15,13 @@ import io.smallrye.reactive.messaging.providers.connectors.WorkerPoolRegistry;
  *
  * Originally generated from: ${templateName} 
  */
-public class TestCdiContext implements CdiContext {
+public class TestCdiContext extends PipelinesCdiContext {
```

Comment:
I: Extend main source class so we don't have to duplicate all bean classes/extensions.